Rakefile

```diff
@@ -40,7 +40,7 @@ task :uninstall do
 end
 
 desc "Build it, tag it and ship it"
-task :ship => :gem do
+task :ship => [:clobber_package, :gem] do
```

Comment:
The :gem task doesn't seem to update/rebuild the gem if one is already present and there are changes.  I found my self constantly rake repackage or rake clobber_package and then rake gem when testing things out.  Is this a fairly common gem/shipping pattern?  I don't actually know if the repackage task is different from the gem task since they both seem to be doing the same thing otherwise (except for the extra cleanup in the beginning).